common/config_parser/config_parser.cpp

```diff
@@ -327,7 +305,7 @@ ExpectedBool MenderConfigFromFile::LoadFile(const string &path) {
 		}
 	}
 
-	/* Vectors/arrays now */
```

Comment:
?

Reply:
They are not in order anymore, that's why I removed this comment and the `Deal with plain string values first` above. It's needed because I have to parse the vectors first for artifact keys and servers, and I didn't want to swap the order of all of them (huge diff, and not really that important).